ghost/admin/app/routes/explore/index.js

```diff
@@ -43,10 +43,22 @@ export default class ExploreRoute extends AuthenticatedRoute {
 
             if (destinationUrl?.includes('/explore')) {
                 isExploreTransition = true;
+                this.explore.isIframeTransition = isExploreTransition;
 
-                // Send the updated route to the iframe
-                if (transition?.to?.params?.sub) {
-                    this.explore.sendRouteUpdate({path: transition.to.params.sub});
+                if (destinationUrl?.includes('/explore/submit')) {
+                    // Ensure the model is loaded before sending the controller action
+                    const model = await this.model;
+                    this.controllerFor('explore').submitExploreSite(model);
+                } else {
+                    let path = destinationUrl.replace(/explore\//, '');
+                    path = path === '/' ? '/explore/' : path;
+
+                    if (destinationUrl?.includes('/explore/about')) {
+                        window.open(`${this.explore.exploreUrl}about/`, '_blank').focus();
```

Comment:
This should simulate the same behaviour as when clicking on the `about` link which opens a new tab.